tool_quality_checks/quality_checks.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+""" TODO Docstring. """
+import os
+import argparse
+import logging
+
+from configparser import RawConfigParser
+from db import ThreediDatabase
+
+logger = logging.getLogger(__name__)
+
+
+def quality_checks(settings):
+    """Overall function for checking our model data"""
+
+    # get database connection
+    db = ThreediDatabase(settings)
+    # TODO: Make it only if install is on.
+    db.initialize_db()
+
+    # get v2_table_names
+    v2_table_names = db.select_table_names("v2%")
+    for table_name in v2_table_names:
+        if db.get_count(table_name) > 0:
```

Comment:
Die `get_count()`: wat doet dat? Count waarvan?
Kan je van de `.get_count() > 0` niet iets als `.is_filled()` maken?

Reply:
Ik wil het aantal objecten in mijn tabel, waarschijnlijk ook ergens terug laten komen in mijn logging in de toekomst. Vandaar dat ik een count query heb gebouwd. is filled zegt alleen ja/nee.